seed/seed.json

```diff
@@ -544,6 +544,38 @@
                 "channel": ["NIGHTLY", "BETA", "RELEASE"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "EpsilonGreedyBanditStudy",
```

Comment:
Add `BraveAds.` as prefix so we append to confirmations

Reply:
👍